inverse_thomson_scattering/fitter.py

```diff
@@ -302,9 +302,9 @@ def _1d_scipy_loop_(config, ts_fitter: TSFitter, previous_weights, batch, tbatch
         method=config["optimizer"]["method"],
         jac=True if config["optimizer"]["grad_method"] == "AD" else False,
         bounds=ts_fitter.bounds,
-        options={"disp": True, "maxiter": 100},
+        options={"disp": True, "maxiter": 1000},
```

Comment:
I think I had set this to `num_epochs` as done on the ADAM side. dont have to do it here but I'll sneak it in eventually if not

Reply:
We definitely should change it to `num_epoch` it was only changed on the adam side of the code, ill add it for the next merge